src/vm.js

```diff
@@ -1053,15 +1053,14 @@ class VM extends common.ServiceObject {
    */
   request(reqOpts, callback) {
     const zone = this.zone;
-    const request = common.ServiceObject.prototype.request;
-    request.call(this, reqOpts, function(err, resp) {
+    super.request(reqOpts, (err, body, res) => {
```

Comment:
@stephenplusplus this was *super* weird.  It looks like the previous callback was accepting the second parameter, naming it and treating it as a response.  But...  as far as I can tell, common always returns `(err, body, response)`.  Any ideas whats up here?